tests/integration/product-type-export.spec.js

```diff
@@ -577,7 +586,7 @@ test(`productType export module
 test(`productType export module
   should output a product types and an attributes file`, (t) => {
   t.timeoutAfter(25000) // 25s
-  before().then(() => {
+  before([null]).then(() => {
```

Comment:
why are you passing a `null` array here? Would passing in no parameter make any difference?

Reply:
Later in the `before` function (which also creates test productTypes) is a condition which removes key from productType draft if the key from `keys` parameter for given productType is `null`. So with this change, I test use case when `productType.key` is not set.